### PR TITLE
Improve SauceLabs iOS setup: iOS 10+

### PR DESCRIPTION
**Pull request by janpio**
### Checklist

- [ ] I've run the tests to see all new and existing tests pass
- [ ] I added automated test coverage as appropriate for this change
- [ ] Commit is prefixed with `(platform)` if this change only applies to one platform (e.g. `(android)`)
- [ ] If this Pull Request resolves an issue, I linked to the issue in the text above (and used the correct [keyword to close issues using keywords](https://help.github.com/articles/closing-issues-using-keywords/))
- [ ] I've updated the documentation if necessary

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -45,6 +46,12 @@ matrix:
     - env: PLATFORM=ios-10.0
       os: osx
       osx_image: xcode9
+    - env: PLATFORM=ios-11.3
+      os: osx
+      osx_image: $XCODE_VERSION
```
Comment (timbru31) [inline .travis.yml]: that does not work :(

Comment (janpio) [reply .travis.yml]: Yep, `osx_image` doesn't seem to like ENV vars. Should be fixed now.

**Comment by timbru31** [comment]
I've opened #122 to fix #119

**Comment by janpio** [comment]
Remaining test failure because of https://github.com/apache/cordova-paramedic/issues/98

**Comment by janpio** [comment]
Reset local build to xcode9 to avoid #98 and similar.

**Pull request merged**